cmd/preflight/cmd/check_container.go

```diff
@@ -257,6 +270,34 @@ func validateCertificationProjectID(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+// checkForNewerReleaseVersion checks if there is a newer release available
+func checkForNewerReleaseVersion(cmd *cobra.Command) {
+	logger := logr.FromContextOrDiscard(cmd.Context())
+
+	// use an authenticated client if a token is provided
+	var client *github.Client
+	ghToken, err := cmd.Flags().GetString("gh-auth-token")
+	if err == nil && len(ghToken) > 0 {
+		client = github.NewClient(&http.Client{
+			// Timeout in 1s in case Github is slow to respond
+			Timeout: time.Second * 1,
+		}).WithAuthToken(ghToken)
+	} else {
+		client = github.NewClient(&http.Client{
+			// timeout in 1s in case Github is slow to respond
+			Timeout: time.Second * 1,
+		})
+	}
+	// check if a newer release is available
+	latestRelease, err := version.Version.LatestReleasedVersion(cmd, client.Repositories)
```

Comment:
This is awkward. LatestReleasedVersion, as @komish stated elsewhere, really doesn't need to be a part of Version.
```suggestion
	latestRelease, err := version.LatestRelease(cmd.Context(), client.Repositories)
```